istio-proxy/istio-proxy.spec

```diff
@@ -130,6 +130,10 @@ TEST_ENVOY=false RUN_TESTS=true %{SOURCE2}
 /usr/local/bin/envoy
 
 %changelog
+* Thu Jun 20 2019 William DeCoste <wdecoste@redhat.com>
```

Comment:
Does it make sense to mention the Istio/Envoy version when specifying a Maistra version upgrade?